integration-tests/jest/jest.spec.js

```diff
@@ -2286,8 +2287,11 @@ describe('jest CommonJS', () => {
         }).catch(done)
       })
     })
-
     it('works with happy-dom', async () => {
+      // happy-dom>=19 can only be used with CJS from node 20 and above
+      if (NODE_MAJOR < 20) {
+        return
+      }
```

Comment:
```suggestion
    (NODE_MAJOR < 20 ? it.skip : it)('works with happy-dom', async () => {
      // happy-dom>=19 can only be used with CJS from node 20 and above
```

Reply:
No, @watson looked into updating tap and it is quite a pain overall due to a complete rewrite.

Instead, we decided we will move to Node.js own test-runner from the moment on we drop support for v5 (Node.js 20 has enough features for us)